GameDefinition/Content/ModeKind.hs

```diff
@@ -177,7 +181,8 @@ hunt = ModeKind  -- melee vs ranged with reaction fire in daylight
       , "* Incapacitate all enemies ASAP"
       ]
   , mdesc   = "Who is the hunter and who is the prey?"
-  , mnote   = "This is yet another reminiscence of the events that led to the long crawl adventure. This episode is quite a tactical challenge, because enemies are allowed to fling their ammo simultaneously at you team, which has no such capacities and focuses on melee combat instead. Act accordingly."
+  , mmotivation = "This is yet another reminiscence of the events that led to the long crawl adventure. This episode is quite a tactical challenge, because enemies are allowed to fling their ammo simultaneously at your team, which has no such capacities and focuses on melee combat instead. Act accordingly."
+  , mhint= ""
```

Comment:
Perhaps I'd try to split it by ending motivation at "capabilities" and putting here "Don't attempt to outshoot the enemy, but focus on melee combat instead.". Or something. The risk being the player would attempt a shooting duel, as in many previous scenarios and would think the challenge is unfair.

Reply:
I'm not sure if I understood where to split it. Also should the "Don't attempt to outshoot the enemy but focus on melee combat instead" go into the motivation as well, or into the mhint field?

Reply:
Oh, sorry, a typo: split at "capacities" (not "capabilities"). I meant this should be the last word of motivation. And hint should be the sentence I cited. Just a rough draft for your consideration.